### PR TITLE
Test PR 2

### PR DESCRIPTION
**Pull request by PranayAgarwal**
This is a test PR to see GitHub's new review features.

### CODE REVIEW

**Comment by PranayAgarwal** [comment]
Top-level comment (not review).

**Comment by pranayagarwal-test** [comment]
User 2 top-level comment (not review).

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
 # test
+This is a test line.
+This is another test line.
+This is another test line.
+
```
Comment (pranayagarwal-test) [inline README.md:5]: This is a grouped review comment #1.

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
 # test
+This is a test line.
+This is another test line.
```
Comment (pranayagarwal-test) [inline README.md:3]: This is a grouped review comment #2.

Comment (pranayagarwal-test) [inline README.md:3]: This is a grouped review comment #3.

Comment (PranayAgarwal) [reply README.md:3]: Addressed

Comment (PranayAgarwal) [reply README.md:3]: Addressed

File: README.md (hunk 1 above)
Comment (PranayAgarwal) [reply README.md:5]: Addressed

**Review by PranayAgarwal — COMMENTED** [review]
Boom.

**Review by pranayagarwal-test — APPROVED** [review]
+1

**Pull request closed without merging**